### DISCUSSION TITLE
Multiple delete keycodes

### DISCUSSION THREAD

**Discussion by ambika-dubey** [xyflow/xyflow#1729] [Ideas]
Is it possible to pass multiple keycodes for delete so that the backspace key and the delete key both delete nodes? I believe right now the `deleteKeyCode` prop only takes one number or string.

**Comment by xplato** [comment]
I don't think ReactFlow supports this directly, but you could just create your own keyboard mapping. I've had success with [react-keyboard-event-handler](https://github.com/linsight/react-keyboard-event-handler) in my project.
